File: src/main/java/seremis/geninfusion/api/soul/IChromosome.java
package seremis.geninfusion.api.soul;

import seremis.geninfusion.util.INBTTagable;

public interface IChromosome extends INBTTagable {

    IAllele<?> getActive();

    IAllele<?> getRecessive();

    IAllele<?> getPrimary();

    IAllele<?> getSecondary();
}

File: src/main/java/seremis/geninfusion/api/soul/IChromosome.scala
package seremis.geninfusion.api.soul

import seremis.geninfusion.util.INBTTagable

trait IChromosome extends INBTTagable {

    def getActive: IAllele

    def getRecessive: IAllele

    def getPrimary: IAllele

    def getSecondary: IAllele
}
